### PR TITLE
fix(node_dump): locate log dir for package installation

### PR DESCRIPTION
**Pull request by qzhuyan**

### CODE REVIEW

**Review by k32 — APPROVED** [review]

**Review by thalesmg — APPROVED** [review]

**Review by thalesmg — APPROVED** [review]
Tiny nit: maybe [this text](https://github.com/emqx/emqx/pull/6187/files#diff-5fa582359e65063fd744f65da47fa8860f5db06fc7938a1baabd27aa518bf6b1R84) still mentions the fixed log dir.

**Review by k32 — DISMISSED** [review]

File: bin/node_dump [hunk 1]
```diff
@@ -1,15 +1,18 @@
 #!/bin/sh
 set -eu
 
+BASE_DIR=$(dirname "$0")
 ROOT_DIR="$(cd "$(dirname "$(readlink "$0" || echo "$0")")"/..; pwd -P)"
+LOG_DIR="$(dirname "$(emqx eval 'hd([ File || #{module := logger_disk_log_h, config := #{file:= File} } <- logger:get_handler_config() ])' | xargs)")"
```
Comment (zmstone) [inline bin/node_dump:6]: What happens if the node is not running?

**Comment by zmstone** [comment]
add this at the top of the script to get all relevant dirs:

```
# shellcheck disable=SC1090
. "$ROOT_DIR"/releases/emqx_vars
```

**Review by zmstone — APPROVED** [review]

**Comment by zmstone** [comment]
same for 4.3 ?

**Pull request merged**